composer.json

```diff
@@ -12,13 +12,14 @@
     "phpunit/phpunit": "^8",
     "plentymarkets/plugin-ceres": "^5.0",
     "plentymarkets/plugin-io": "^5.0",
-    "plentymarkets/plugin-interface": "dev-stable7"
+    "plentymarkets/plugin-interface": "dev-stable7",
+    "findologic/findologic-api": "1.8.0"
```

Comment:
1. This package is not not for dev purposes
2. It should replace the http_request2 package, please also remove that one